agenta-backend/agenta_backend/config.toml

```diff
@@ -4,4 +4,10 @@ database_url="sqlite:////db/deploy.db"
 redis_url="redis://redis:6379/0"
 docker_hub_url="https://hub.docker.com/v2/repositories/{}/{}"
 docker_hub_repo_owner="agentaai"
-docker_hub_repo_name="templates"
\ No newline at end of file
+docker_hub_repo_name="templates"
+feature_flag="oss"
+openai_api_key="value"
+ecr_repository="value"
+aws_secret_access_key="value"
```

Comment:
please don't add variables to the config.toml use instead env variables as defined in docker-compose (see how we use the feature flag as env var as example)